recommended-rules.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: contribute
+title: Recommended Rules
+---
+
+# Recommended Rules
+
+In the documentation, we provide a list of
+[recommended rules](https://docs.bazel.build/versions/master/rules.html).
+
+This is a set of high quality rules, which will provide a good experience to our
+users. We make a distinction between the supported rules, and the hundreds of
+rules you can find on the Internet.
```

Comment:
Can we elaborate on the distinctions here? e.g. are supported rules also recommended by default? Are bazelbuild/rules_* supported / recommended by default?

Reply:
There's no special case for rules under `bazelbuild/rules_`. Hopefully most of them will qualify.